webclient/src/components/elements/ProjectDialog.vue

```diff
@@ -65,16 +68,6 @@ export default {
 </script>
 
 <style scoped>
-input, textarea {
-  display: block;
```

Comment:
moved these input related styles to Hacksite.vue so that they can be reused across the app.